content/07.pathogenesis.md

```diff
@@ -79,3 +79,17 @@ There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and
 However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
 Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
 Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
+Using such databases, the systems level reconstruction of the PPI (Protein-Protein Interaction) enabled the generation of hypotheses on the mechanism of action of SARS-CoV-2 and suggested drug targets.
+
+#### Protein-protein interaction networks
+
+In a first study [@doi:10.1101/2020.03.22.002386], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells and 332 high confidence human proteins that interact with them were consequently identified. 
+Notably, this study suggested the interaction of SARS-CoV-2 with innate immuty pathways, with a similarity to *Mycobacterium tuberculosis* host-pathogen interactome.
```

Comment:
This is not incorrect but somehow misleading... 

SARS-CoV-2 protein and human protein interactome study indicates several targets with known roles in innate immunity. based on similarities to interactomes derived for 10 other pathogens, WNV and MTb are most similar to SARS-CoV-2. MTb is not the top hit but it is interesting due to lung related symptoms in tb patients. right? I think it can be a little better clarified.

Reply:
ok, I tried to come up with different wording in 4556380. Please let me know if this makes sense.